pkg/rainbow/storage/dbram/dbram.go

```diff
@@ -12,14 +12,24 @@ import (
 
 // DB client.
 type DB struct {
-	options []rainbow.Option
+	options  []rainbow.Option
+	cpFormat rainbow.CPFormat
 }
 
 func (db *DB) InsertOptions(options []rainbow.Option) error {
 	db.options = options
 	return nil
 }
 
+func (db *DB) InsertCPFormat(table rainbow.CPFormat) error {
+	db.cpFormat = table
+	return nil
+}
```

Comment:
Do we still need this methods ? I thought we said we do not store `CPFormat` in the database.

Reply:
Oops, you are right. I remove them. :wink: